<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsPage</class>
 <widget class="QWidget" name="AppearanceSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="layout_appearancesettingspage">
   <item>
    <widget class="QGroupBox" name="groupbox_colors">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="layout_colors">
      <item>
       <widget class="QRadioButton" name="use_system_color_set">
        <property name="text">
         <string>&amp;Use the system default color set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_a_custom_color_set">
        <property name="text">
         <string>Use a custom color set</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_foreground_color">
        <item>
         <widget class="QLabel" name="select_foreground_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select foreground color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_foreground_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_background_color">
        <item>
         <widget class="QLabel" name="select_background_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select background color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_background_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_background_image">
     <property name="title">
      <string>Background image</string>
     </property>
     <layout class="QVBoxLayout" name="layout_background_image">
      <item>
       <widget class="QRadioButton" name="use_default_background">
        <property name="text">
         <string>Default bac&amp;kground image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_no_background">
        <property name="text">
         <string>&amp;No background image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_album_cover_background">
        <property name="toolTip">
         <string>The album cover of the currently playing song</string>
        </property>
        <property name="text">
         <string>Albu&amp;m cover</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_custom_image">
        <item>
         <widget class="QRadioButton" name="use_custom_background_image">
          <property name="text">
           <string>Custom image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="background_image_filename">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_background_image_filename_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_custom_background_image_options" native="true">
        <layout class="QVBoxLayout" name="layout_custom_background_image_options">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_custom_position" native="true">
           <layout class="QHBoxLayout" name="layout_custom_position">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_position">
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combobox_backgroundimageposition">
              <item>
               <property name="text">
                <string>Upper Left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Upper Right</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Middle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom Left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom Right</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Max cover size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinbox_background_image_maxsize">
              <property name="maximum">
               <number>9000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer_position">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_background_image_stretch" native="true">
           <layout class="QHBoxLayout" name="layout_background_image_stretch">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="checkbox_background_image_stretch">
              <property name="text">
               <string>Stretch background image to fill playlist background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkbox_background_image_do_not_cut">
              <property name="text">
               <string>Do not cut image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkbox_background_image_keep_aspect_ratio">
              <property name="text">
               <string>Keep aspect ratio</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="layout_blur_opacity">
           <item row="0" column="0">
            <widget class="QLabel" name="select_background_blur_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Blur amount</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="blur_slider">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="background_blur_radius_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0px</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="select_opacity_level_label">
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="opacity_slider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="background_opacity_label">
             <property name="text">
              <string>40%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_system_icons">
     <property name="text">
      <string>Use system theme icons</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
